### ISSUE TITLE
Please provide FCOS OVA image for VMware on aarch64/MacOS

### ISSUE THREAD

**Issue by martinhinze** [coreos/fedora-coreos-tracker#1594]
### Describe the enhancement

Currently there is no OVA image provided for using FCOS on VMware and aarch64 with an ignition file on the project's [download page](https://fedoraproject.org/coreos/download/?stream=stable&arch=aarch64#download_section). The FCOS [user guide](https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-vmware/) advises using an OVA image for provisioning FCOS in VMware systems. This is especially important for doing this in an automated way with shell scripts.

### System details

VMware Fusion on aarch64/MacOS

**Comment by patrickb777** [comment]
I'm also looking for an arm64 FCOS ova.

**Comment by fifofonix** [comment]
In the past I have occasionally had the need to spin up local VMs especially when investigating/reproducing issues with `next`, or investigating new functionality, or contributing to test days. An arm64 FCOS ova would help given I'm using VMWare Fusion although I know I have other virtualization options open to me ... and perhaps should investigate ...

**Comment by jbtrystram** [comment]
This was discussed today in the community meeting. 

It was agreed that we will build vmware aarch64 artifacts and test them out. If all goes well we will enable it in the pipeline and publish those. 

item list from @dustymabe 
 - someone does a local build and test on mac/vmware fusion (@fifofonix may be willing to do the test if someone else does the build)
 -  add the entry to the artifact list in the pipeline
 -  make sure it shows up on the website and in the builds browser

@fifofonix was volunteer'ed for testing :)

**Comment by fifofonix** [comment]
I just had a look at the `fedora-coreos-pipeline` project for the first time. If I understand things correctly I should be able to make a one line change in the `config.yaml` linked above in a forked version of the project. Then I would be able to trigger a `developer` build of one of the streams using the `deploy` tooling. This of course would necessitate a configured pipeline processing infrastructure *or* using the existing core project infrastructure by applying for access with a sponsor etc. Have I understood the steps roughly? Half hoping I could have a crack at this this coming weekend if we could sort the access out quickly.

**Comment by jbtrystram** [comment]
yes that's about it. Maybe you could also build the artifact manually using coreos-assembler ?

**Comment by fifofonix** [comment]
Sounds like it is time for me to try `coreos-assembler`!

**Comment by jlebon** [comment]
It sounds totally fine to me to enable it in a non-production stream first (e.g. rawhide) and then you can test the artifact that comes out of that: https://github.com/coreos/fedora-coreos-pipeline/pull/960

Building it with cosa is cool too if you'd like to do that, but the most valuable part for us to unblock this is actually checking if the artifact works. :)

**Comment by fifofonix** [comment]
I am unable to get either of the last two days' VMWare OVA artifacts for aarch64 to work with VMWare Fusion 13.5.0 on an arm64 MacOS machine. The boot does not even begin with Fusion believing that the OVA in question is actually to be x86 as opposed to arm64. The precise message is: 

<img width="281" alt="image" src="https://github.com/coreos/fedora-coreos-tracker/assets/37665762/2574d7ba-8426-4b15-84da-281f61d6bc75">

I actually used two different methods for launching a VMWare Fusion VM and both had the same result (one of the methods was that documented in the FAQ).

I also went back to an x86 machine and confirmed that when selecting an x86 artifact that still worked as expected (it has been a long time since I've created a VM this way). No issues there so I feel pretty sure we're getting a real failure. Over to you chaps on why this might be the case, or for other tests you'd like me to run.

**Comment by dustymabe** [comment]
Maybe there is something in the vmware ovf/ova template file that needs to specify what architecture the disk image is? the template we then modify is located here: https://github.com/coreos/coreos-assembler/blob/main/src/vmware-template.xml

**Comment by fifofonix** [comment]
Wanted to run the VMWare Fusion test for Fedora40 test week but that is still not possible absent an arm64 OVA.

**Comment by dustymabe** [comment]
we reverted it because it appeared additional work would be needed to make it actually work. I can try to give you a one-off build, but I don't anticipate it would have magically just started working without someone identifying why it wasn't working to begin with and fixing it. 

LMK if you want that one-off build.

**Comment by fifofonix** [comment]
Got it. No need for a on-off build. Thanks @dustymabe

**Comment by dustymabe** [comment]
Now, don't get me wrong. We'd love to fix it and get it delivered. Let me know if anyone finds out the reason and we can try to get it added.
